cylc/flow/network/resolvers.py

```diff
@@ -686,19 +686,22 @@ async def _mutation_mapper(
         self, command: str, kwargs: Dict[str, Any], meta: Dict[str, Any]
     ) -> Optional[Tuple[bool, str]]:
         """Map between GraphQL resolvers and internal command interface."""
+
+        log_msg = f"[command] {command}"
+        user = meta.get('auth_user', self.schd.owner)
+        if user != self.schd.owner:
+            log_msg += (f" (issued by {user})")
+        LOG.info(log_msg)
```

Comment:
This message was previously only logged if:

* The command was not issued by the user.
* The command was not `put_messages` (#3647)

Reply:
Yes that statement is correct. (But what's you're point?)

Now it is logged:

- regardless of user, but "issued by {user}" is added if not the owner
  - this is more consistent: log the message, but add the extra info is necessary
    
- for all commands including `put_message`
  - were we singling this command out because (a) we don't want to log it because it is so frequent; and/or (b) because it is "normal" for the job owner to be different due to the owner simply having a different user name on the job platform? 
  - my thinking was, other users could in principle issue `cylc message` commands and so it should be logged, and all commands should be logged equally
  
(However, I did pose the question in the issue description, in case discussion was needed).

Reply:
Fair enough, how about only log `put_message` commands if they were issued from another user account?

Reply:
Note this fix goes beyond `put_messages`. Previously this message was not logged at all for commands (such as `stop`) that don't have wrapper methods in the resolver.

But I'll tweak the fix for the `put_message` case...

Reply:
Done.